integration_test/third_party_apps_test.go

```diff
@@ -565,7 +578,18 @@ func TestThirdPartyApps(t *testing.T) {
 			for attempt := 1; attempt <= 4; attempt++ {
 				logger := gce.SetupLogger(t)
 				logger.ToMainLog().Println("Calling SetupVM(). For details, see VM_initialization.txt.")
-				vm := gce.SetupVM(ctx, t, logger.ToFile("VM_initialization.txt"), gce.VMOptions{Platform: tc.platform, MachineType: agents.RecommendedMachineType(tc.platform)})
+				options := gce.VMOptions{
```

Comment:
mega-nit: does this need a space?

```suggestion
				options := gce.VMOptions {
```

Reply:
it definitely doesn't need one in order to compile. I think it's a stylistic choice; i can't find any official rule about this. But I think people tend to write structs without the extra whitespace.